### PR TITLE
Added missing/invalid/inaccurate docblocs to all methods of Phly_Couch, removed require_once for Zend's Json class

### PR DESCRIPTION
**Pull request by laurynas-karvelis**
Added missing/invalid/inaccurate docblocs to all methods of Phly_Couch, removed require_once for Zend's Json class

### CODE REVIEW

File: Phly_Couch/library/Phly/Couch.php [hunk 1]
```diff
@@ -382,28 +383,28 @@ public function docSave($document, $id = null, $db = null)
...
-        if (null !== $id) {
+        if(null !== $id) {
             $method = 'PUT';
-            $path  .= $id;
+            $path .= $id;
         }
 
-        if (is_string($document)) {
-            if ('{' != substr($document, 0, 1)) {
+        if(is_string($document)) {
```
Comment (weierophinney) [inline Phly_Couch/library/Phly/Couch.php]: There should be a space between the "if" and the opening paren in all cases.

File: Phly_Couch/library/Phly/Couch/Document.php [hunk 2]
```diff
@@ -32,20 +46,34 @@ public function setId($id)
         return $this;
     }
 
+    /**
+     * Gets the Id of the document
+     *
+     * @param void
+     * @return string|null
+     */
+
```
Comment (weierophinney) [inline Phly_Couch/library/Phly/Couch/Document.php]: There should be no empty line between the docblock and the method in all cases.

**Comment by weierophinney** [comment]
Please rebase off  of current master, and make the changes as requested, and I'll merge.

Thanks!

**Comment by laurynas-karvelis** [comment]
Will do in couple of days :)

Best regards,

_Laurynas Karvelis_

The basement is overcrowded but the top floor is empty.

Mobile: +44 7713 167071
Website: http://www.explosive-brains.com
LinkedIn profile: http://uk.linkedin.com/in/laurynaskarvelis
About.me profile: http://about.me/karvelis

On 21 May 2012 17:34, Matthew Weier O'Phinney <
reply@reply.github.com

**Pull request closed without merging**